src/classes/HiFiAudioAPIData.ts

```diff
@@ -445,15 +451,26 @@ export class HiFiAudioAPIData {
      * ❌ The server never sends `userRolloff` data.
      */
     userRolloff: number;
+
+    /**
+     * This represents a queue of other user gains that the client wants to be adjusted the next time data is sent to the server, which apply to the current user only.
+     * This can be used to provide a more comfortable listening experience for the client. However, setting a gain here does not on its own change the gain of another user. See instead {@link HiFiCommunicator.setOtherUserGainForThisConnection}, which allows you to set the desired gain for a user and also send the data to the server. If you need to perform moderation actions which apply to all users, use the {@link https://docs.highfidelity.com/rest/latest/index.html|Administrative REST API}.
+     * 
+     * Each key is the hashed visit ID of a user whose gain will be affected, and its associated value is the desired gain of the other user. Any gain set here only affects the current user's session. Other users beside the current user will not perceive a difference in volume when a gain setting is set here.
+     * 
+     * These gains are not persistent. If the other user's session restarts, then their gain will be reset for the current user. If the current user's session restarts, then all other user gains will be reset.
+     */
+    otherUserGainQueue: OtherUserGainMap;
```

Comment:
See above. Is there a use case for allowing developers to interact with this directly? I think it might make more sense to hide this from devs (and therefore from the constructor parameters) and just keep it private and initialize it in the ctor. Developers should (I think?) _just_ use the `setOtherUserGainForThisConnection` method rather than trying to understand what's going on with the gain queue.

Reply:
You're right, I don't think there's a use case for using this directly. Removing the ability to set it in the constructor seems reasonable. But we can't make it private because it's used by `HiFiMixerSession` to sync the other user gains. One alternative option is to have a separate class called `InternalHiFiAudioAPIData` which stores data like this. But for now, keeping this data all in the same place seems better.

Reply:
Yep, that's fine for now, thank you! ❤️